services/mailer/mailer.go

```diff
@@ -67,6 +70,11 @@ func (m *Message) ToMessage() *gomail.Message {
 		msg.SetBody("text/plain", plainBody)
 		msg.AddAlternative("text/html", m.Body)
 	}
+
+	if len(msg.GetHeader("Message-ID")) == 0 {
+		msg.SetHeader("Message-ID", "<"+m.generateMessageID()+">")
+		log.Warn("Mail to %s with subject %s did not contain a Message-Id, I set one for you.")
```

Comment:
These warning doesn't seem to help users (or developers)

If we want to indicate that the message id is auto-generated, we can make the message-id like: `autogen-UUID@domain`

Reply:
ok, i prefixed the generated id with `autogen` :)